WORKSPACE.yaml

```diff
@@ -7,7 +7,7 @@ defaultArgs:
   jbMarketplacePublishTrigger: "false"
   publishToJBMarketplace: true
   localAppVersion: unknown
-  codeCommit: 0175d2fd7b619fb1c34dcf2a0e17a909acd1be3e
+  codeCommit: a229ac8c8ea6f9461a5cb306d3177765b8836641
```

Comment:
This is no longer needed https://github.com/gitpod-io/gitpod/pull/8313#issuecomment-1047921031